backend/util.go

```diff
@@ -0,0 +1,10 @@
+// Package backend contains all the storage system backends.
+package backend
+
+// StringPointerOrNil returns a pointer to the string if non-empty or nil if it is empty
+func StringPointerOrNil(s string) *string {
```

Comment:
Not sure where this refactor crept in from - it doesn't seem relevant to this PR?

Utilities for use by backends go in `lib/` - maybe `lib/backend` - though not sure a 4 line function merits a whole new package!

Reply:
This was related to the new B2 tests. I've backed it out of this PR.